grpc/src/test/java/io/stargate/grpc/payload/ProcessResultTests.java

```diff
@@ -46,32 +46,31 @@
   @ParameterizedTest
   @MethodSource("results")
   public void processResult(
```

Comment:
The class name ends in `Tests` (with an `s`)... I'm just wondering if maven picks up and actually executes this test :thinking: By convention unit tests have names without the `s`.

Reply:
Thanks, typo.